tox.ini

```
@@ -50,6 +50,7 @@ commands =
 deps =
     aiohttp
     urllib3
+    git+https://github.com/juju/python-libjuju.git@master # wokeignore:rule=master
```

sigh... how do you feel about referencing a commit instead?

```suggestion
    git+https://github.com/juju/python-libjuju.git@34981c59eac8db89320bf0067bebd2547da12d45
```

just to be sure here -- i'm OKAY with this

---

+1